lesson6Khanh/src/main/java/com/trainingdeveloperpro/androidk001/lesson5/nguyenquockhanh/studentlist/AddFragment.kt

```diff
@@ -0,0 +1,22 @@
+package com.trainingdeveloperpro.androidk001.lesson5.nguyenquockhanh.studentlist
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+const val NAME : String = "name"
```

Comment:
Nên đổi tên biến lại cho chuẩn coding convention
```suggestion
const val EXTRA_STUDENT_NAME : String = "name"
```